riscv-regs/src/regs.rs

```diff
@@ -117,3 +117,19 @@ impl GeneralPurposeRegisters {
 #[derive(Default)]
 #[repr(C)]
 pub struct FloatingPointRegisters([u64; 32]);
+
+/// The vector register file. We don't expect to directly interact with a guest's vector state
+/// other than for saving/restoring the registers, so simply treat the register file as an array
+/// of 256b values. This actually depends on the vlenb csr, so if the register is greater than 256
+/// bits (i.e. the vlenb csr is greater than 32) we will need to increase this.
+
+// the width of a vector register in u64's
+pub const MAX_VLENB_DIV8: usize = 4;
```

Comment:
This is slightly confusing since 256 / sizeof(u64) is 32, so perhaps it's that divided again by 8? A good trick might be to reuse the same const fn() trick from up above to compute the value so that the intent is obvious (and values can simply be adjusted in the function if needed in the future).

Reply:
fixed